Makefile

```diff
@@ -75,7 +78,7 @@ endif
 
 FINCH_CORE_DIR := $(CURDIR)/deps/finch-core
 
-remote-all: arch-test finch install.finch-core-dependencies finch.yaml networks.yaml config.yaml
+remote-all: arch-test finch install.finch-core-dependencies finch.yaml networks.yaml config.yaml $(OUTDIR)/finch-daemon/finch@.service
```

Comment:
Do we really need separate targets for `networks.yaml`, `config.yaml`,  `$(OUTDIR)/finch-daemon/finch@.service`?

Reply:
I think anything that writes a file should probably have its own rule so that its not re-created unless the file is deleted